modules/farm_fd2/src/entrypoints/direct_seeding/direct_seeding.submission.e2e.cy.js

```diff
@@ -81,7 +118,43 @@ describe('Direct Seeding: Submission tests', () => {
      * lib.submitForm function that is called when the "Submit"
      * button is clicked.
      */
-    cy.intercept('POST', '**/api/log/activity', cy.spy().as('submitSpy'));
+    cy.intercept('POST', '**/api/asset/plant', (req) => {
+      // check plant name
+      cropMap = maps.get('crop');
+      let cropID = cropMap.get('ARUGULA').id;
+      expect(req.body.data.relationships.plant_type.data[0].id).to.eq(cropID);
+    }).as('submitSpy');
```

Comment:
In order to use the `submitSpy` later to check the number of times the intercept happened (e.g. `cy.get('@submitSpy').should('have.callCount', 3)`) then you will need to give each one of them a different alias (e.g. maybe call this one `plantAssetSpy`)